### ISSUE TITLE
Can't call method "fulltext" on an undefined value

### ISSUE THREAD

**Issue by chorny** [Leont/App-ModuleBuildTiny#29]
I'm running `mbtiny dist` on https://github.com/monsieurp/Gentoo-App-Pram.git and receive error:

```
Changes appears to be empty
Can't call method "fulltext" on an undefined value at C:/strawberry514/perl/site/lib/App/ModuleBuildTiny.pm line 51.
```

**Comment by Leont** [comment]
This shouldn't happen, and somehow I can't reproduce this easily though I can see how it could happen. Does 16bf190d fix this for you?

**Comment by chorny** [comment]
> Does 16bf190 fix this for you?

yes, it does

**Comment by Leont** [comment]
> yes, it does

Released as 0.017

**Comment by monsieurp** [comment]
What a coincidence! I ran into the same error a couple of minutes, looked up the error and found this bug report mentioning my own package ;) I just released a new version of Gentoo-App-Pram thanks to your fix! Many thanks @Leont!
